assets/js/src/BoardDragAndDrop.js

```diff
@@ -103,7 +103,6 @@ Kanboard.BoardDragAndDrop.prototype.refresh = function(data) {
     this.app.hideLoadingIcon();
     this.executeListeners();
     this.dragAndDrop();
-    KB.tooltip();
```

Comment:
Only this one needs to be removed?? the `tooltip()` function is already executed when the DOM is ready.

Reply:
No, I removed the function name and changed it so the whole tooltip section gets executed at dom ready. So no need for a separate function call/name anymore.
If I kept the function name then your suggestion would be sufficient too, but I don't like unneeded function names :-)